ads/google/a4a/performance.js

```diff
@@ -158,23 +112,39 @@ export class GoogleAdLifecycleReporter extends BaseLifecycleReporter {
   constructor(win, element, namespace, correlator, slotId) {
     super();
 
-    this.QQID_HEADER = 'X-QQID';
+    /** @private {!Window} @const */
     this.win_ = win;
+
+    /** @private {!Element} @const */
     this.element_ = element;
+
+    /** @private {string} @const */
     this.namespace_ = namespace;
+
+    /** @private {number} @const */
     this.slotId_ = slotId;
+
+    /** @private {number} @const */
     this.correlator_ = correlator;
+
+    /** @private {string} @const */
     this.slotName_ = this.namespace_ + '.' + this.slotId_;
-    this.qqid_ = null;
-    this.initTime_ = Date.now();
+
+    /** @private {number} @const */
+    this.initTime_ = getTimingDataSync(win, 'navigationStart') || Date.now();
```

Comment:
Are we worried about fallback to using Date.now causing unknown variance in the data?  Could we at least include in the request an indicator that we were not able to use navigationStart?

I suppose in the end all of the deltas are based on an arbitrary starting point so it doesn't really matter but if that's true, then why use navigationStart at all?